estimagic/tests/visualization/test_comparison_plot.py

```diff
@@ -0,0 +1,332 @@
+import json
+from os.path import join
+
+import numpy as np
+import pandas as pd
+import pandas.testing as pdt
+import pytest
+
+from estimagic.visualization.comparison_plot import _add_dodge_and_binned_x
+from estimagic.visualization.comparison_plot import _create_bounds_and_rect_widths
+from estimagic.visualization.comparison_plot import _determine_figure_height
+from estimagic.visualization.comparison_plot import _determine_plot_heights
+from estimagic.visualization.comparison_plot import _df_with_all_results
+from estimagic.visualization.comparison_plot import _df_with_color_column
+from estimagic.visualization.comparison_plot import _df_with_plot_specs
+from estimagic.visualization.comparison_plot import _find_next_lower
+from estimagic.visualization.comparison_plot import _find_next_upper
+from estimagic.visualization.comparison_plot import _flatten_dict
+from estimagic.visualization.comparison_plot import _prep_result_df
+from estimagic.visualization.comparison_plot import (
+    _reset_index_without_losing_information,
+)
+from estimagic.visualization.comparison_plot import MEDIUMELECTRICBLUE
+
+FIX_PATH = "estimagic/tests/visualization/comparison_plot_fixtures/"
```

Comment:
Checkout ``pathlib`` which is amazing for cross-platform paths.
```
from pathlib import Path
from estimagic.visualization import comparison_plot


FIX_PATH = Path(comparison_plot.__path__, "comparison_plot_fixtures")
```
Then, you do not need ``os.path.join``. It is simply ``path = FIX_PATH / "minimal_res_dict.json"`` and if you need to have a string (most python functions are fine by accepting a path) do ``str(path)``.

Reply:
much more concise this way, too! Thanks for the suggestion!

I think with __ attributes ``__file__`` would be correct. However, [this SO answer](https://stackoverflow.com/a/6209894) suggests using the ``inspect`` module is more robust for getting the path to the current module so I use that. 

Adopted in 1e0580d06bc44eada85c4a4b7b3ff810264fda82.